MedscribeUI/src/components/NoteControls/NotecontrolsLayout.tsx

```diff
@@ -0,0 +1,78 @@
+import { Select, Button } from '@mantine/core';
+import { useState } from 'react';
+import ThreeBtnSelector from '../Utilities/ThreeBtnSelector';
+
+const visitSelectorLabel = 'Visit';
+const defaultPronoun = 'He';
+
+function NoteControlsLayout() {
+  const [selectedPronoun, setSelectedPronoun] = useState(defaultPronoun);
+  const [selectedNoteLength, setSelectedNoteLength] = useState('Standard');
```

Comment:
nit: lets remove this we aren't going to include it in the mvp for now. I know what it looks like on the example and thats my bad for not clarifying!

Reply:
Removed NoteLength